### DISCUSSION TITLE
163

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#111] [General]
# 潮流周刊第163期 - 买了麻将

封面图来源于上周买的一个麻将，摸着温润如玉的感觉很是喜欢，今年春节打算把家里接到杭州过年，麻将会是一个特别适合过年的活动。

https://weekly.tw93.fun/posts/163-%E4%B9%B0%E4%BA%86%E9%BA%BB%E5%B0%86

**Comment by lainbo** [comment]
人在中国大陆的还是不建议在flags中打开QUIC, 因为大陆的isp限制udp很严格, 一旦网站支持h3基本就是负优化, 比如开了这个选项后打开Cloudflare的dashboard会非常的慢, 很多套用了Cloudflare服务的网页也会变慢

**Reply by lainbo** [reply]
如果你在打开后没有出现类似的情况那可真是太好了, 如果出现了变慢的情况记得先考虑这个因素

**Comment by shiguiyou** [comment]
显卡截图的工具是啥，求。

**Reply by tsingwong** [reply]
应该是作者之前推荐的 tinySnap

**Reply by tw93** [reply]
https://shots.so/
https://shottr.cc/
最近用这两个

**Comment by markyun** [comment]
周刊的RSS名称，显示有点问题

**Reply by tw93** [reply]
方便截图看看吗？具体是啥问题，我使用的 NetNewsWire 效果是这样
![](https://cdn.fliggy.com/upic/zYE52J.png)

**Comment by michaelliuyang** [comment]
虚实之间的播客，强推，无废话，主题明确，在轻松的氛围下，拓展了视野和边界

**Reply by haohaolalahao** [reply]
还有类似的博客推荐么？

**Reply by tw93** [reply]
试试从这个里面找找 https://podtown.xyz/

**Reply by haohaolalahao** [reply]
谢谢大佬！

**Comment by Suremotoo** [comment]
岭上开花！
![咲](https://p26-sign.bdxiguaimg.com/tos-cn-i-0000/4d834e62e3b145e283abd48cda0ed0fe~tplv-pk90l89vgd-crop-center:864:486.jpeg?appId=3586&channelId=0&customType=custom%2Fnone&from=401_large_image_list&imageType=video1609&isImmersiveScene=0&is_stream=0&logId=2023111819462241B2043249E974B49CFE&requestFrom=401&x-expires=1731843983&x-signature=5QLEfqwWA5gENqqxoZgQl3XnhJ0%3D)

**Reply by tw93** [reply]
哈哈哈哈  666666
